src/symbolic/symbolic_wasm_ffi.ml

```diff
@@ -42,6 +44,12 @@ module M :
 
   let symbol_f64 () = Choice.with_new_symbol (Ty_fp 64) Expr.symbol
 
+  let symbol_range (lo : Value.int32) (hi : Value.int32) =
+    let* x = symbol_i32 () in
+    let* () = assume (Value.I32.le lo x) in
+    let* () = assume (Value.I32.gt hi x) in
+    Choice.return x
```

Comment:
Here too you can use `let open Choice in` and remove the `return` by using `let+` on the last let-binding.